config/config.toml.example

```diff
@@ -418,20 +418,9 @@ store-liveness-timeout = "5s"
 ttl-refreshed-txn-size = 33554432
 
 [tikv-client.copr-cache]
-# Whether to enable the copr cache. The copr cache saves the result from TiKV Coprocessor in the memory and
-# reuses the result when corresponding data in TiKV is unchanged, on a region basis.
-enable = true
-
-# The capacity in MB of the cache.
+# The capacity in MB of the cache. Zero means to disable coprocessor cache.
 capacity-mb = 1000.0
 
-# Only cache small requests. Zero means no limits for requests.
-admission-max-ranges = 500
-# Only cache requests whose result set is small.
-admission-max-result-mb = 10.0
-# Only cache requests takes notable time to process.
-admission-min-process-ms = 5
```

Comment:
Seems that some users are tend to modify these configurations, do we need to remove them?

Reply:
It's a request from the PM.

Reply:
@XuHuaiyu PTAL.

Reply:
@bb7133 may have impression about the client that uses these configuration values. If we drop them, then we cannot serve this kind of clients any more.

Reply:
How about:
1. only expose `capacity-mb` to users by default to increase usability.
2. make other config items configurable, but hidden by default.

Reply:
> How about:
> 
> 1. only expose `capacity-mb` to users by default to increase usability.
> 2. make other config items configurable, but hidden by default.

Hide configurations looks good to me.